lib/logto_client.dart

```diff
@@ -55,22 +71,26 @@ class LogtoClient {
     return token?.serialization;
   }
 
+  // Get the idToken claims.
```

Comment:
no need for this since the function name itself do the same job?

Reply:
cool removed